deltachat-ffi/src/lib.rs

```diff
@@ -59,9 +63,12 @@ use deltachat::chatlist::Chatlist;
 // TODO: constants
 
 // dc_context_t
-
+//
+// This is newtyped so that cbindgen does not need to scan dependencies and so that we can
+// have our own doc comment.
```

Comment:
No feedback, just curious: Why do we want different doc comments for the C API and for the Rust API?

Reply:
They are vastly different in how you use them, e.g. you need to talk about different type names `dc_context_t` vs `Context` and the ways of using them are very different too, e.g. implicit drop vs `dc_context_unref()`.  Basically the FFI needs to get the docs that are currently in the `deltachat.h` file.  I think if the FFI docs start talking about `Context.get_config()` or such things it would get very confusing for FFI users.

Reply:
rustdoc has better output than Doxygen, in rustdoc the search is way better to use:
https://c.delta.chat
https://cffi.delta.chat/deltachat/ (currently has nearly no doc comments so is not that useful yet)